spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -70,3 +70,50 @@ definitions:
             desc: |
               PRN-1 identifier of the satellite signal for which
               acquisition was attempted
+
+ - MSG_ACQ_SV_PROFILE:
+    id: 0x001E
+    short_desc: Reacq perfomance measurement and debug
+    desc: |
+      The message describes SV profile during acquisition time.
+      The message is useful to debug and measure the performance of
+      the new (re-)acq design.
+    public: true
+    fields:
+        - N:
+            type: u8
+            desc: Number of SVs included in the message
+        - job_type:
+            type: u8
+            desc: SV search job type (deep, fallback, etc)
+        - int_time:
+            type: u8
+            units: ms
+            desc: Acquisition integration time
+        - sid_code:
+            type: u8
+            desc: Signal constellation, band and code
+        - bin_width:
+            type: u16
+            units: Hz
+            desc: Acq frequency bin width
+        - timestamp:
+            type: u32
+            units: ms
+            desc: Timestamp of the job complete event
+        - time_spent:
+            type: u32
+            units: us
+            desc: Time spent to search for sid.code
+        - cf_min:
+            type: u32
+            units: Hz
+            desc: Doppler range lowest frequency
+        - cf_max:
+            type: u32
+            units: Hz
+            desc: Doppler range highest frequency
+        - cf:
+            type: u32
+            units: Hz
+            desc: Doppler value of detected peak
```

Comment:
@denniszollo Agree, CN0 would be useful and needs to be added. There is already a message MSG_ACQ_RESULT I updated the re-acq design document accordingly.